docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,58 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start an Interledger node locally on your computer and connect to an external ILP testnet.
+
+This tutorial describes how to:
+
+1. Configure [`moneyd`](https://github.com/interledgerjs/moneyd) to connect to a node on testnet
+2. Start an [Interledger node](https://github.com/interledgerjs/ilp-connector) locally using moneyd
+3. Send value to a [payment pointer](https://interledger.org/rfcs/0026-payment-pointers/) on the testnet
+
+## Before you begin
+For this tutorial, you will need `moneyd` and SPSP installed on your computer. You can learn how to install `moneyd` 
+and SPSP from the [Getting Started](getting-started.md) tutorial.
+
+NOTE: For this tutorial you do not need to use real cryptocurrency. Since you will be running ILP on a testnet, 
+funds will be used from a [testnet faucet](https://xrpl.org/xrp-test-net-faucet.html) and have no real-world value.
+
+## Configuring moneyd
+To configure moneyd: 
+
+1. Open a terminal and use the following command:
+```shell
+moneyd xrp:configure --testnet --advanced
+```
+
+2. When prompted for the `BTP host of parent connector`, enter:
+```shell
+xpring.dev
+```
+
+3. Press enter for other default options.
+
+## Starting moneyd
+After you’ve configured `moneyd`, run the following command to start an Interledger node on your computer and 
+connect it to a routing node on the testnet:
```

Comment:
>start an Interledger node on your computer and connect it to a routing node on the testnet:

start an ILP node on your computer and connect it to another ILP node on testnet

Reply:
https://github.com/interledger/interledger.github.io/pull/108/commits/040b4c13cc77219844de0e840d34a66e26207ce1#diff-95b449acc2e692458b91372a5b6e33e7R35